app/controllers/concerns/hyrax/manages_embargoes.rb

```diff
@@ -0,0 +1,21 @@
+module Hyrax
+  module ManagesEmbargoes
+    extend ActiveSupport::Concern
+
+    included do
+      include Hyrax::ThemedLayoutController
+      with_themed_layout '1_column'
```

Comment:
Does this imply the install generator overrides this? i.e.
- https://github.com/samvera-labs/hyrax/blob/master/lib/generators/hyrax/install_generator.rb#L40

That generator _seems_ to apply this to a blacklight layout, so does that apply to this too?

Are you wanting both lines 6 & 7 removed or just line 7?